.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Build
+on: [push, workflow_dispatch]
+jobs:
+  build:
+    name: ${{ matrix.label }}-${{ matrix.arch }}-${{ matrix.build_type }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-latest
+            label: windows
+            arch: x64
+            triplet: x64-windows
+            build_type: release
+          - os: windows-latest
+            label: windows
+            arch: arm64
+            triplet: arm64-windows
+            build_type: release
+          - os: ubuntu-latest
```

Comment:
Can you please restore ` ubuntu-22.0`. I wanna keep that in-sync this the Linux image we use in DOSBox Staging to avoid weird compatibility problems.